docs/reference/configuration/data_types.rst

```diff
@@ -72,167 +61,352 @@ This section only applies if a fixed seed is used:
-and hard limits 0 and 10:
+The ``TRUNCATED_NORMAL`` keyword is utilized to assign a truncated normal distribution to a variable.
+This distribution works as follows:
 
+   1. Draw a random variable :math:`X \sim N(\mu,\sigma)`.
+   2. Clamp :math:`X` to the interval [min, max].
+
+It's important to note that this is **not** a proper normal distribution due to the constraints applied.
+Thus, the act of clamping values to the `[min,max]` interval should be an exceptional event. 
+
+Syntax
+~~~~~~
 ::
 
-   VAR3  TRUNCATED_NORMAL  1  0.25   0  10
+  VAR TRUNCATED_NORMAL <mean> <standard_deviation> <min> <max>
 
+Parameters
+~~~~~~~~~~
+- **<mean>**: Specifies the mean of the normal distribution prior to truncation.
+- **<standard_deviation>**: Indicates the standard deviation of the distribution before truncation.
+- **<min>**: Determines the lower truncation limit.
+- **<max>**: Establishes the upper truncation limit.
+
+Example
+~~~~~~~
+Reasonable usage of this distribution where the number of values being clamped is relatively low.
+::
 
-UNIFORM
-,,,,,,,
+   VAR TRUNCATED_NORMAL 2 0.7 0 4
 
-A stochastic variable is uniformly distributed if has a constant
-probability density on a closed interval. Thus, the uniform
-distribution is completely characterized by it's minimum and maximum
-value. To assign a uniform distribution to a variable, use the keyword
-UNIFORM, which takes a minimum and a maximum value for a the
-variable. Here is an example, which assigns a uniform distribution
-between 0 and 1 to a variable ``VAR4``:
+.. image:: fig/truncated_ok.png
 
+Example showing excessive clamping. This should be avoided.
 ::
 
-   VAR4 UNIFORM 0 1
+   VAR TRUNCATED_NORMAL 0 1 -1 1
 
-It can be shown that among all distributions bounded below by a and
-above by b, the uniform distribution with parameters a and b has the
-maximal entropy (contains the least information). Thus, the uniform
-distribution should be your preferred prior distribution for robust
-modeling of bounded variables.
+.. image:: fig/truncated_bad.png
 
+UNIFORM: Uniform Distribution
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-LOGUNIF
-,,,,,,,
+The ``UNIFORM`` keyword is used to assign a uniform distribution to a variable.
+A variable is considered uniformly distributed when it has a constant probability density over a closed interval.
+Thus, the uniform distribution is fully characterized by it's minimum and maximum values.
 
-A stochastic variable is log uniformly distributed if it's logarithm
-is uniformly distributed on the interval [a,b]. To assign a log
-uniform distribution to a a variable, use the keyword LOGUNIF, which
-takes a minimum and a maximum value for the output variable as
-arguments. The example
+Syntax
+~~~~~~
+::
+
+  VAR UNIFORM <min_value> <max_value>
+
+Parameters
+~~~~~~~~~~
+- **<min_value>**: Specifies the lower bound of the uniform distribution.
+- **<max_value>**: Sets the upper bound of the uniform distribution.
```

Comment:
Start with "The". I would prefer the same with parameters for the distributions below as well.

Reply:
Done